### PR TITLE
Registering application

### PR DESCRIPTION
**Pull request by scripstace**
Hi @Brit200313, please review and merge if appropriate.

Thanks

### CODE REVIEW

File: app/assets/javascripts/registered_applications.coffee [hunk 1]
```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```
Comment (Brit200313) [inline app/assets/javascripts/registered_applications.coffee]: If you are not going to use this stylesheet, please delete it!

File: app/assets/stylesheets/registered_applications.scss [hunk 2]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the RegisteredApplications controller here.
```
Comment (Brit200313) [inline app/assets/stylesheets/registered_applications.scss]: Same thing - if you are not using it, please delete it.

File: app/helpers/registered_applications_helper.rb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+module RegisteredApplicationsHelper
```
Comment (Brit200313) [inline app/helpers/registered_applications_helper.rb]: Remove please.

File: spec/features/user_adds_registered_application_spec.rb [hunk 4]
```diff
@@ -0,0 +1,33 @@
+require 'rails_helper'
```
Comment (Brit200313) [inline spec/features/user_adds_registered_application_spec.rb]: Nice job on these tests.

File: test/models/registered_application_test.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
```
Comment (Brit200313) [inline test/models/registered_application_test.rb]: I'm assuming you will test this model in the future.

File: app/assets/stylesheets/application.scss [hunk 6]
```diff
@@ -27,3 +31,20 @@
 .jumbotron {
```
Comment (Brit200313) [inline app/assets/stylesheets/application.scss]: any thoughts on switching this up? I know you have design skills so I'd love to see your Bloc portfolio vary up a bit.

**Comment by Brit200313** [comment]
@scripstace I left several notes on this pull request. When you are ready for me to review again, please comment bellow and @ my name. Thanks!

**Comment by scripstace** [comment]
@Brit200313, I deleted the unecessary file. Please merge if appropriate.

Thanks

**Pull request merged**